tools/flake8/config.ini

```diff
@@ -1,6 +1,8 @@
+[flake8]
 filename = *.py,*.py.in
 show-source = True
 statistics = True
 count = True
-ignore = F403
-max-line-length = 79
+ignore = F401,F403,F405
```

Comment:
* `F401` is for unused imports. Why disable it? People should remove unused imports.
* `F405` is mostly triggered for start references. This is to encourage people only adding the exact functions they need to the global module. Why is it disabled?